src/lsst/cst/conversions/tools.py

```diff
@@ -51,214 +50,93 @@ def data_id_to_str(data_id: dict):
     Returns
     -------
     data_id_str: `str`
-        Data identifier string
+        Data identifier string.
     """
-    cal_exp_id = CalExpId(**data_id)
-    return str(cal_exp_id)
+    data_id_str = json.dumps(data_id)
+    return data_id_str
 
 
-def sort_dataframe(
-    df: pd.DataFrame,
-    sort_key: str,
-    ascending: bool = False,
-    set_index: bool = True,
-) -> pd.DataFrame:
-    """Return a sorted copy of the dataframe 'df' by index,
-    selecting the desired order (ascending or descending)
-    using the ascending argument, it also exchanges the index
-    of the dataframe for the sort_key if set_index parameter
-    is True (default).
+def psf_size_at_pixel_xy(psf, bbox, xy: (int, int)) -> dict:
+    """Obtains the size of the PSF in an image
+    at a given xy coordinate.
 
     Parameters
     ----------
-    df : `pandas.DataFrame`
-        dataframe to be sorted
-    sort_key: `str`
-        column key used to sort the dataframe
-    ascending: `bool`, optional
-        ascending/descending sorting
-    set_index: `bool`, optional
-        set sorted key as the dataframe index
+    psf : `lsst.meas.extensions.psfex.PsfexPsf` or
```

Comment:
```suggestion
    psf : `lsst.meas.extensions.psfex.PsfexPsf` or \
```
Needs the backslash to pull in the next line (but I think the next line needs rewriting in that case).

Reply:
I can add it but... maybe did you miss it's in the """docstring"""? Not part of the function input definition. It's just descriptive.

Reply:
Yes, I was commenting on the numpydoc

```
Parameters
-----------
psf : `some.long.type.that.is.very.long` or \
        `some.other.long.type.that.would.go.overly.long`
    Documentation about the parameter.
```
The `or` at the end suggested to me that your `psf` parameter took two different types so you wanted those types to be listed in the numpydoc as types rather than having the second one as part of the description text.

Reply:
Oh I see, adding a backslash just there will help in this special case -- thanks for explaining. Have added.

```
    Parameters
    ----------
    psf : `lsst.meas.extensions.psfex.PsfexPsf` or \
          `lsst.meas.algorithms.CoaddPsf`
        PSF object from a calexp or deepCoadd respectively; use .getPsf().
```